requirements.txt

```diff
@@ -1,6 +1,5 @@
-sphinx==2.4.5
+sphinx==7.1.2
 sphinx-rtd-theme==0.4.3
 sphinxcontrib-plantuml==0.15
 sphinxcontrib-httpdomain==1.7.0
-docutils==0.14
-sphinx-tabs==3.3.1
```

Comment:
Can we get rid of `sphinx-tabs`? That's what gives us tabs in our current docs.

Reply:
They are still there even with this removed. I thought the same but building the docs with the image has them working fine.

Reply:
I can't replicate. After building this image (and updating it a little) I hit this error:

![image](https://github.com/OSC/ood-documentation-build/assets/4874123/42102dac-d3f6-4fef-bb1b-6c20cbe0fe51)